ci/test_integrationtests.sh

```diff
@@ -17,8 +17,20 @@ export LD_LIBRARY_PATH=$BUILD_DIR/depends/$HOST/lib
 
 cd build-ci/dashcore-$BUILD_TARGET
 
+if [ "$SOCKETEVENTS" = "" ]; then
+  # Let's switch socketevents mode to some random mode
+  R=$(($RANDOM%2))
+  if [ "$R" == "0" ]; then
+    SOCKETEVENTS="select"
+  else
+    SOCKETEVENTS="poll"
+  fi
+fi
+echo "Using socketevents mode: $SOCKETEVENTS"
+EXTRA_ARGS="--dashd-arg=-socketevents=$SOCKETEVENTS"
+
 set +e
-./test/functional/test_runner.py --ci --coverage --failfast --nocleanup --tmpdir=$(pwd)/testdatadirs $PASS_ARGS
+./test/functional/test_runner.py --ci --coverage --failfast --nocleanup --tmpdir=$(pwd)/testdatadirs $PASS_ARGS $EXTRA_ARGS
```

Comment:
Soo, does each run of CI randomly choose a different one?

Reply:
Yes, every dashd from that one run (including all tests) will use the same setting